packages/iobridge-commons/package.json

```diff
@@ -11,6 +11,9 @@
     "build": "tsc"
   },
   "dependencies": {
-    "@ckb-lumos/codec": "^0.17.0-rc10"
+    "@ckb-lumos/codec": "^0.17.0-rc10",
+    "@ckb-lumos/lumos": "^0.17.0-rc10",
+    "@metamask/eth-sig-util": "^4.0.1",
+    "ethereumjs-util": "^7.1.4"
```

Comment:
i found `ethereumjs-util` only imported in tests, we should move it to `devDependencies`

Reply:
boom in ci, we need to ignore `*.spec.ts` for (no-unpublished-import)[https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-unpublished-import.md#convertpath]